### ISSUE TITLE
Release 2.3.1 breaking installs

### ISSUE THREAD

**Issue by bjacobel** [jquery/sizzle#386]
2.3.1 was released earlier this hour, but we're unable to install it from the npmjs.org registry.

```
$ > npm install sizzle@2.3.1
...
npm ERR! fetch failed http://registry.npmjs.org/sizzle/-/sizzle-2.3.1.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
npm ERR! fetch failed http://registry.npmjs.org/sizzle/-/sizzle-2.3.1.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 404
npm ERR! fetch failed http://registry.npmjs.org/sizzle/-/sizzle-2.3.1.tgz
npm ERR! Darwin 15.5.0
npm ERR! argv "/Users/bjacobel/.nvm/versions/node/v4.4.3/bin/node" "/Users/bjacobel/.nvm/versions/node/v4.4.3/bin/npm" "install" "sizzle@2.3.1"
npm ERR! node v4.4.3
npm ERR! npm  v2.15.1

npm ERR! fetch failed with status code 404
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/bjacobel/code/edx-platform/npm-debug.log
```

**Comment by bjacobel** [comment]
Here's the output of `npm info sizzle`:

```javascript
{ name: 'sizzle',
  description: 'A pure-JavaScript, bottom-up CSS selector engine designed to be easily dropped in to a host library.',
  'dist-tags': { latest: '2.3.1' },
  versions:
   [ '1.0.0',
     '1.0.1',
     '1.1.0',
     '2.0.0',
     '2.1.0',
     '2.1.1',
     '2.2.0',
     '2.2.1',
     '2.3.0',
     '2.3.1' ],
...
```

**Comment by mgol** [comment]
I'm observing the same. Weird.

**Comment by gibson042** [comment]
We just published 2.3.2 and unpublished 2.3.1. Thanks for the report, and please let us know if you experience any more problems.

**Comment by mgol** [comment]
2.3.2 published, 2.3.1 unpublished. It should be fine now.

Thanks for the quick report!
